src/components/common/searchfield.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import RightArrowIcon from "../../assets/right_arrow.svg";
+import { InputText } from "primereact/inputtext";
+import { Button } from 'primereact/button';
+import { Image } from 'primereact/image';
+
+type Props = {
+    searchValue: string;
+    setSearchValue: (a: string) => void;
+};
+
+const SearchField: FC<Props> = ({ searchValue, setSearchValue }) => {
+
+    return (
+        <div className="flex sm:hidden px-6 py-3 border-slate-200  border-solid	 border-b justify-between">
+            <InputText type="text" name="q" value={searchValue} onChange={(e) => setSearchValue(e.target.value)} className="w-full focus:outline-none placeholder:text-stone-500 bg-transparent placeholder:font-helvetica placeholder:font-medium" placeholder="Search Order" />
+            <Button type="submit" className='-ml-8'>
```

Comment:
`-ml-8`
I dont think this is the correct class name.
Please check once.

Reply:
Its margin-left:-2rem